subprojects/docs/docs.gradle

```diff
@@ -263,6 +263,7 @@ def dslStandaloneDocbook = tasks.register("dslStandaloneDocbook", UserGuideTrans
     javadocUrl = '../javadoc'
     dsldocUrl = '../dsl'
     websiteUrl = 'https://gradle.org'
+    tags = ["noNavBar"]
```

Comment:
I think we need to keep this out so the noNavBar types don't show up in the table either.